test/test_jit.py

```diff
@@ -1727,6 +1727,11 @@ def forward(self, x):
         FileCheck().check("mul").check("add").run(str(trace))
 
     def test_trace_c10_ops(self):
```

Comment:
would something like `@skipIf(hasattr(torch.ops._caffe2, 'GenerateProposals')` here work instead? it's more idiomatic

Reply:
I tried that... actually, we did some magic in __set_state__/__get_state__, hasattr(torch.ops._caffe2, 'GenerateProposals') always return false...

Reply:
It'd be nice to fix hasattr to work (probably because we throw RuntimeError instead of AttributeError), but it's non-blocking